src/pages/SignIn.tsx

```diff
@@ -0,0 +1,178 @@
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* form side */}
+      <div className="md:w-1/2 flex flex-col justify-center lg:px-36 px-10 py-10">
+        <h1 className="text-2xl font-bold mb-2 text-dark-green">Login</h1>
+        <p className="text-base border-gray-300 text-gray-400 mb-2">
+          Welcome back to your account
+        </p>
+        <Formik
+          initialValues={{ email: "", password: "" }}
+          validationSchema={Yup.object({
+            email: Yup.string()
+              .email("Invalid email address")
+              .required("Required"),
+            password: Yup.string()
+              .min(6, "Too Short!")
+              .max(50, "Too Long!")
+              .required("Required"),
+          })}
+          onSubmit={(values, { setSubmitting }) => {
+            setTimeout(async () => {
+              setLoading(true);
+              const resultAction = await dispatch(
+                login({
+                  email: values.email,
+                  password: values.password,
+                  device_id: window.location.hostname,
+                })
+              );
+              if (login.fulfilled.match(resultAction)) {
+                setErrortext("");
+                setLoading(false);
+                setSubmitting(false);
+                navigate("/dashboard");
+              } else {
+                if (resultAction.payload) {
+                  setErrortext(resultAction.payload.message);
+                }
+                setErrortext(resultAction.payload.message);
+              }
+              setLoading(false);
+              setSubmitting(false);
+            }, 400);
+          }}
+        >
+          <Form className="flex flex-col justify-center">
+            <label
+              htmlFor="email"
+              className="mb-2 mt-6 font-base text-dark-green font-semibold"
+            >
+              Email
+            </label>
+            <Field
+              name="email"
+              type="email"
+              className="focus:shadow-outline w-full  appearance-none rounded-md border border-gray-300 p-3 leading-tight text-gray-700 focus:outline-none text-sm"
+              placeholder="Please Enter Your Email"
+            />
+            <ErrorMessage name="email">
+              {(msg) => <div className="my-1 text-red-500 text-sm">{msg}</div>}
+            </ErrorMessage>
+
+            <label
+              htmlFor="password"
+              className="mb-2 mt-6 font-base text-dark-green font-semibold"
+            >
+              Password
+            </label>
+            <Field
+              name="password"
+              type="passowrd"
+              className="focus:shadow-outline w-full appearance-none rounded-md border border-gray-300 p-3 leading-tight text-gray-700 focus:outline-none text-sm"
+              placeholder="Please Enter Your Password"
+            />
```

Comment:
update the value of your type to `type="password"`

Reply:
Sure, I realized that and I have corrected it in the logout pr